Framework/Algorithms/test/PDCalibrationTest.h

```diff
@@ -475,6 +477,104 @@ class PDCalibrationTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(calTable->cell<double>(index + 2, 1), calTable->cell<double>(index, 1), 1E-5); // det 102
     TS_ASSERT_DELTA(calTable->cell<double>(index + 3, 1), calTable->cell<double>(index, 1), 1E-5); // det 103
   }
+
+  void test_exec_ikeda_carpenter() {
+    // test the algorithm using the IkedaCarpenterPV peak function
+    const double ref_difc = 2208.287616521762;
+
+    const std::vector<double> dValues{0.3117, 0.3257, 0.3499, 0.4205, 0.4645, 0.4768, 0.4996, 0.5150, 0.5441, 0.5642,
```

Comment:
We can probably just do the last four peaks, `dvalues = (.8920,1.0758,1.2615,2.0599)`